frame/staking/runtime-api/src/lib.rs

```diff
@@ -0,0 +1,33 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2019-2022 Parity Technologies (UK) Ltd.
```

Comment:
```suggestion
// Copyright (C) 2023 Parity Technologies (UK) Ltd.
```

Same with other files.